pages/index.js

```diff
@@ -0,0 +1,63 @@
+import Link from 'next/link';
+import React from 'react';
+
+import Banner from '../components/Banner';
+import Carousel from '../components/Carousel';
+import Committees from '../components/CommitteeSpread';
+import Layout from '../components/Layout';
+import Article from '../components/NewsArticle';
+import SocialMedia from '../components/SocialMedia';
+import TGBanner from '../components/TechGala/ViewBanner';
+import data from '../data';
+
+function Home () {
+	const { carousel, committees, news } = data;
+	return (
+		<Layout>
+			<div className="home-page text-center">
+				<TGBanner />
+				<Banner />
+				<div className="content-section">
+					<h2>The largest Computer Science community at UCLA</h2>
+					<p className="subheading">ACM at UCLA is the largest computer science student organization in Southern California. We welcome students of all backgrounds, interests, and skill levels to join our community and share our love for technology. ACM is comprised of eight committees — each serving a different topic and mission. Our events are open to everyone, regardless of major or experience!</p>
+					<SocialMedia />
+				</div>
+				<br />
+				<Committees committees={committees} />
+				<div className="button-section">
+					{ /** CHANGE THIS LINK TO ABOUT PAGE LATER */ }
+					<Link href="/committees"><a className="button">Learn More About Our Committees</a></Link>
```

Comment:
What does this comment mean?

Reply:
Good catch, I think I added it before refactoring how `.button` worked.